Public/Src/Engine/Scheduler/Tracing/Log.cs

```diff
@@ -154,6 +154,19 @@ internal static string PipWriteFileFailedMessage(string pipDescription, string p
             string moniker,
             string message);
 
+        [GeneratedEvent(
+            (ushort)EventId.PipIpcFailedDueToInfrastructureError,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Error,
+            Keywords = (int)(Keywords.UserMessage | Keywords.InfrastructureError),
+            EventTask = (ushort)Tasks.PipExecutor,
+            Message = "IPC operation '{operation}' could not be executed via IPC moniker '{moniker}' because of an infrastructure error. Error: {message}")]
+        internal abstract void PipIpcFailedDueToInfrastructureError(

```

Comment:
> You need to add the "UserError" keyword here if your new event is the one we're going to use for infrastructure/internal errors. Otherwise these will keep getting marked as internal errors which is the default.

The idea was the following:
  - `PipIpcFailed` continues to be classified as internal error
  - the existing `PipIpcFailedDueToInvalidInput` is used for IPC user errors
  - this new `PipIpcFailedDueToInfrastructureError` is used for IPC infra errors